tests/cases/fourslash/commentsLinePreservation.ts

```diff
@@ -129,16 +129,16 @@ verify.quickInfos({
     3: ["(parameter) param1: string", "first line of param\n\nparam information third line"],
 
     g: ["function g(param1: string): void", "This is firstLine\nThis is second Line"],
-    4: ["(parameter) param1: string", "param information first line"],
+    4: ["(parameter) param1: string", " param information first line"],

```

Comment:
Notice how this tests changed. Arguably this should be the right behavior. Let me know if you think otherwise.